docs/reference/nuspec.md

```diff
@@ -87,18 +87,22 @@ A URL for the package's home page, often shown in UI displays as well as nuget.o
 > [!Important]
 > licenseUrl is being deprecated. Use license instead.
 
-A URL for the package's license, often shown in UI displays as well as nuget.org.
+A URL for the package's license, often shown in UI displays such as nuget.org.
```

Comment:
UI displays sounds weird to me :D 

What is this sentence trying to communicate? 
My first thoguh twas that it was trying to say that it's displayed in the NuGet PM UI + feeds like nuget.org

Reply:
Agreed that it sounds weird. I believe it's exactly what you said: PM UI + nuget.org. Nothing came to mind to improve the wording here, and my changes were already drastic enough. Let me know if you have any suggestions, otherwise I'll leave it as is.

Reply:
I've changed `UI displays` to just `UIs`. Let me know what you think